schema/adminschema/v1.json

```diff
@@ -96,77 +96,129 @@
-                  "httpMethod": "POST",
-                  "path": "admin",
-                  "request": {
-                      "$ref": "Admin"
-                  },
-                  "response": {
-                      "$ref": "Admin"
-                  }
-              }
+    "Admin": {
+      "methods": {
+        "Get": {
+          "id": "dex.admin.Admin.Get",
+          "description": "Retrieve information about an admin user.",
+          "httpMethod": "GET",
+          "path": "admin/{id}",
+          "parameters": {
+            "id": {
+              "type": "string",
+              "required": true,
+              "location": "path"
+            }
+          },
+          "parameterOrder": [
+            "id"
+          ],
+          "response": {
+            "$ref": "Admin"
+          }
+        },
+        "Create": {
+          "id": "dex.admin.Admin.Create",
+          "description": "Create a new admin user.",
+          "httpMethod": "POST",
+          "path": "admin",
+          "request": {
+            "$ref": "Admin"
+          },
+          "response": {
+            "$ref": "Admin"
+          }
+        }
+      }
+    },
+    "State": {
+      "methods": {
+        "Get": {
+          "id": "dex.admin.State.Get",
+          "description": "Get the state of the Dex DB",
+          "httpMethod": "GET",
+          "path": "state",
+          "response": {
+            "$ref": "State"
           }
-      },
-      "State": {
-          "methods": {
-              "Get": {
-                  "id": "dex.admin.State.Get",
-                  "description": "Get the state of the Dex DB",
-                  "httpMethod": "GET",
-                  "path": "state",
-                  "response": {
-                      "$ref": "State"
-                  }
-              }
+        }
+      }
+    },
+    "Client": {
+      "methods": {
+        "Create": {
+          "id": "dex.admin.Client.Create",
+          "description": "Register an OpenID Connect client.",
+          "httpMethod": "POST",
+          "path": "client",
+          "request": {
+            "$ref": "ClientCreateRequest"
+          },
+          "response": {
+            "$ref": "ClientCreateResponse"
           }
-      },
-      "Client": {
-          "methods": {
-              "Create": {
-                  "id": "dex.admin.Client.Create",
-                  "description": "Register an OpenID Connect client.",
-                  "httpMethod": "POST",
-                  "path": "client",
-                  "request": {
-                      "$ref": "ClientCreateRequest"
-                  },
-                  "response": {
-                      "$ref": "ClientCreateResponse"
-                  }
-              }
+        }
+      }
+    },
+    "Connectors": {
+      "methods": {
+        "Set": {
+          "id": "dex.admin.Connector.Set",
+          "description": "Set the connectors for the dex system.",
+          "httpMethod": "PUT",
```

Comment:
You generally always set all the connectors at once. We can split this out into the full CRUD if you'd like and have them deal with individual connectors (maybe based on ID).